### PR TITLE
[memgraph-2.12 < E-mage-path-ivan] Add docs for path

### PR DESCRIPTION
**Pull request by imilinovic**
### Description

Add documentation for the path MAGE module.

### Pull request type

Please check what kind of PR this is:

- [X] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
[PR394](https://github.com/memgraph/mage/pull/394)
(especially necessary if the PR is related to a release)

Closes:
(paste the link to the issue it closes)

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [x] If release-related, add release note on product PR

### CODE REVIEW

**Comment by imilinovic** [comment]
Matija ended up having no changes on this module so thats why I'm instantly merging to 2.12.

File: pages/advanced-algorithms/available-algorithms/path.mdx [hunk 1]
```diff
@@ -21,12 +21,107 @@ The `path` module allows users to explore different paths, filter relationships,
...
++------------------------------------------------------------------------------------------------------------------+
+| result                                                                                                           |
++------------------------------------------------------------------------------------------------------------------+
+| [(:Node1), [:CONNECTED], (:Node2), [:CONNECTED], (:Node3), [:CONNECTED], (:Node4), [:CONNECTED], (:Node5)]       |
++------------------------------------------------------------------------------------------------------------------+
+```
+
+### `combine()`
+
+Combines the two given paths into one. The end of the first path and the beginning of the second path must match. Throws an exception if the given paths can't be combined.
```
Comment (vpavicic) [inline pages/advanced-algorithms/available-algorithms/path.mdx:62]:
```suggestion
Combines two given paths into one. The end of the first path and the beginning of the second path must match. Throws an exception if the given paths can't be combined.
```

File: pages/advanced-algorithms/available-algorithms/path.mdx [hunk 2]
```diff
@@ -21,12 +21,107 @@ The `path` module allows users to explore different paths, filter relationships,
...
+
+- `Path` ➡ The subpath of the given path.
+
+{<h4> Usage: </h4>}
+
+```cypher
+CREATE (:Node1)-[:CONNECTED]->(:Node2)-[:CONNECTED]->(:Node3)-[:CONNECTED]->(:Node4)-[:CONNECTED]->(:Node5);
+```
+```cypher
+MATCH path = (:Node1)-[:CONNECTED*4]->(:Node5) RETURN path.slice(path, 1, -1) AS result;
```
Comment (vpavicic) [inline pages/advanced-algorithms/available-algorithms/path.mdx:111]:
```suggestion
MATCH path = (:Node1)-[:CONNECTED*4]->(:Node5)
RETURN path.slice(path, 1, -1) AS result;
```

File: pages/advanced-algorithms/available-algorithms/path.mdx [hunk 3]
```diff
@@ -21,12 +21,107 @@ The `path` module allows users to explore different paths, filter relationships,
...
+```
+
+The result will be given in shortened form with () signifying a node and [] signifying a relationship.
+```plaintext
```
Comment (vpavicic) [inline pages/advanced-algorithms/available-algorithms/path.mdx:115]:
```suggestion

```plaintext
```

File: pages/advanced-algorithms/available-algorithms/path.mdx [hunk 4]
```diff
@@ -21,12 +21,107 @@ The `path` module allows users to explore different paths, filter relationships,
...
+
+```cypher
+CREATE (:Node1)-[:CONNECTED]->(:Node2)-[:CONNECTED]->(:Node3)-[:CONNECTED]->(:Node4)-[:CONNECTED]->(:Node5)-[:CONNECTED]->(:Node6);
+```
+```cypher
+MATCH (node1:Node1), (node4:Node4), (node6:Node6) MATCH path1 = (node1)-[:CONNECTED*3]->(node4) MATCH path2 = (node4)-[:CONNECTED*2]->(node6) RETURN path.combine(path1, path2) AS result;
+```
+
+The result will be given in shortened form with () signifying a node and [] signifying a relationship.
+```plaintext
```
Comment (vpavicic) [inline pages/advanced-algorithms/available-algorithms/path.mdx:83]:
```suggestion

```plaintext
```

File: pages/advanced-algorithms/available-algorithms/path.mdx [hunk 5]
```diff
@@ -21,12 +21,107 @@ The `path` module allows users to explore different paths, filter relationships,
...
+
+```cypher
+CREATE (:Node1)-[:CONNECTED]->(:Node2)-[:CONNECTED]->(:Node3)-[:CONNECTED]->(:Node4)-[:CONNECTED]->(:Node5);
+```
+```cypher
+MATCH path = (:Node1)-[:CONNECTED*4]->(:Node5) RETURN path.elements(path) AS result;
+```
+
+The result will be given in shortened form with () signifying a node and [] signifying a relationship.
+```plaintext
```
Comment (vpavicic) [inline pages/advanced-algorithms/available-algorithms/path.mdx:52]:
```suggestion

```plaintext
```

**Pull request merged**